book/src/configuration.md

```diff
@@ -2,8 +2,10 @@
 
 > **Note:** The configuration file is unstable and may be deprecated in the future.
 
-Some lints can be configured in a TOML file named `clippy.toml` or `.clippy.toml`. It contains a
-basic `variable = value` mapping e.g.
+Some lints can be configured in a TOML file named `clippy.toml` or `.clippy.toml`, placed in a directory specified by
+the environment variable `CLIPPY_CONF_DIR`, or if that's not found, the environment variable
```

Comment:
I would put this in a list, so

```
- `CLIPPY_CONF_DIR` ...; or
- ...; or
- ...
```

instead of in text. Would make it easier to read.